Makefile

```diff
@@ -65,10 +65,19 @@ lint-with-docker:  ## Lint source code with docker image of golangci-lint
 .PHONY: lint lint-with-local lint-with-docker
 
 # --- Docker -------------------------------------------------------------------
+TAG ?= $(error tag not set)
```

Comment:
Perhaps "TAG" is a bit generic - it could also mean a git tag. Since the target is a "docker" target, perhaps it should be `DOCKER_TAG`? And the error message perhaps should say `DOCKER_TAG not set`

Reply:
ok, done, also updated in release.yaml.